api/openapi.yaml

```diff
@@ -1558,6 +1558,404 @@ components:
         infoFinder:
           $ref: '#/components/schemas/InfoFinderConfig'
 
+    EffectiveAssetScanConfig:
+      type: object
+      description: Effective asset scan configuration
+      properties:
+        effectiveScanFamiliesConfig:
+          $ref: '#/components/schemas/EffectiveScanFamiliesConfig'
+
+    EffectiveScanFamiliesConfig:
+      type: object
+      description: The effective scan families configuration
+      properties:
+        sbom:
+          $ref: '#/components/schemas/EffectiveSBOMConfig'
+        vulnerabilities:
+          $ref: '#/components/schemas/EffectiveVulnerabilitiesConfig'
+        malware:
+          $ref: '#/components/schemas/EffectiveMalwareConfig'
+        rootkits:
+          $ref: '#/components/schemas/EffectiveRootkitsConfig'
+        secrets:
+          $ref: '#/components/schemas/EffectiveSecretsConfig'
+        misconfigurations:
+          $ref: '#/components/schemas/EffectiveMisconfigurationsConfig'
+        exploits:
+          $ref: '#/components/schemas/EffectiveExploitsConfig'
+        infoFinder:
+          $ref: '#/components/schemas/EffectiveInfoFinderConfig'
+
+    EffectiveScanFamiliesTemplate:
+      type: object
+      description: Common fields of scan families config
+      properties:
+        enabled:
+          type: boolean
+        scannerList:
+          type: array
+          items:
+            type: string
+        inputs:
+          type: array
+          items:
+            $ref: '#/components/schemas/EffectiveInput'
+
+    EffectiveSBOMConfig:
```

Comment:
I think we should align the SBOM and vulnerability structures to be the same as the other families:
```
      allOf:
        - $ref: '#/components/schemas/EffectiveScanFamiliesTemplate'
        - type: object
          properties:
            <Specific family config>
```
            
We should change also the .families.yaml structure IMO.

Maybe even better to change the structure of a family config in .families.yaml into (example for secrets):
```
secrets:
  metadata:
    enabled: false
    scanners_list:
      - "gitleaks"
    inputs:
      - input: "./"
        input_type: "dir"
  scanners_config:
    gitleaks:
      binary_path: "/usr/local/bin/gitleaks"

```

That way, we can reuse the same metadata object also in odata.go